ResearchKit/Common/ORKOrderedTask.h

```diff
@@ -475,6 +475,7 @@ typedef NS_OPTIONS(NSUInteger, ORKPredefinedTaskOption) {
                                       localized text is displayed.
  @param distanceInMeters            The timed walk distance in meters.
  @param timeLimit                   The time limit to complete the trials.
+ @param turnAroundTimeLimit         The time limit to complete the turn around step.
```

Comment:
@vtourraine , @YuanZhu-apple 
Yes, should also include allowing the turn-around step to be skipped. I hadn't realized that there wasn't a check for `turnAroundTimeLimit == 0`. Additionally, original signature should be included. Deprecated, maybe, but apps that use this module should not break.